src/test/java/org/terasology/logic/health/BlockTest.java

```diff
@@ -33,7 +33,7 @@
 import static org.junit.Assert.assertTrue;
 
 public class BlockTest extends ModuleTestingEnvironment {
-    private static final Vector3i BLOCK_LOCATION = Vector3i.zero().add(Vector3i.down());
+    private static final Vector3i BLOCK_LOCATION = new Vector3i().add(0, -1, 0);
```

Comment:
As @keturn pointed out before, the `Vector3i.zero()` was explicit in what the vector is, whereas `new Vector3i()` only implicitly holds that information, and might not be clear from looking at the test case. 

```suggestion
    private static final Vector3i BLOCK_LOCATION = new Vector3i(0, 0, 0).add(0, -1, 0);
```

Reply:
I also suggest using Vector3ic.

Reply:
incorporated Michael's comment in the suggestion 😉